src/OnlineSales/Entities/File.cs

```diff
@@ -0,0 +1,29 @@
+﻿// <copyright file="File.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using OnlineSales.DataAnnotations;
+
+namespace OnlineSales.Entities
+{
+    [Table("files")]
```

Comment:
@Mixer57 table name must be in singular form, i.e. "file"

Reply:
fixed